src/modules/authentication/passport/oidc.strategy.spec.ts

```diff
@@ -145,13 +152,99 @@ describe('OpenIdConnectStrategy', () => {
 
         it('should call super.authenticate with options', () => {
             const request: Request = createMock<Request>();
-            request.query['requiredStepUpLevel'] = StepUpLevel.GOLD;
+            request.session.requiredStepupLevel = StepUpLevel.GOLD;
             sut.authenticate(request);
 
             expect(superPassportSpy).toHaveBeenCalledWith(request, { acr_values: 'gold' });
         });
     });
 
+    const mockTime = (time: number): jest.SpyInstance => jest.spyOn(Date.prototype, 'getTime').mockReturnValue(time);
+
```

Comment:
If possible, I'd prefer using jests fake-timers instead of mocking the Date object